### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by Martha-K**

### CODE REVIEW

File: src/pages/HomeStyles.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export const StyledSlide = styled.img`
 `;
 
 export const Hero = styled.div`
-  margin-top: 30px;
+  /* margin-top: 30px; */
```
Comment (gennadiy123) [inline src/pages/HomeStyles.js]: remove comment please

File: src/pages/Products.tsx [hunk 2]
```diff
@@ -1,49 +1,66 @@
...
+  Div,
+  ProductName,
+  CardInfoWrapper,
+  ProductWrapping,
+} from "./ProductsStyles";
+import { mockProduct } from "../mockProducts";
 
 export const Products = () => {
+  const [isArmature, setIsArmature] = useState(true);
+  console.log("isArmature", isArmature);
```
Comment (gennadiy123) [inline src/pages/Products.tsx]: remove console.log please

File: src/pages/HomeStyles.js (hunk 1 above)
Comment (Martha-K) [reply src/pages/HomeStyles.js]: Done

File: src/pages/Products.tsx (hunk 2 above)
Comment (Martha-K) [reply src/pages/Products.tsx]: Done

**Comment by Martha-K** [comment]
Done

**Comment by Martha-K** [comment]
Done

**Pull request merged**